src/NuGet.Clients/NuGet.PackageManagement.UI/LoadingStatusIndicator.cs

```diff
@@ -81,5 +76,36 @@ public void SetError(string message)
             Status = LoadingStatus.ErrorOccurred;
             ErrorMessage = message;
         }
+
+        internal string LocalizedStatus
+        {
+            get
+            {
+                switch (Status)
+                {
+                    case LoadingStatus.Loading:
+                        return LoadingMessage;
+
+                    case LoadingStatus.NoItemsFound:
+                        return Resources.Text_NoPackagesFound;
+
+                    case LoadingStatus.Cancelled:
+                        return Resources.Status_Canceled;
+
+                    case LoadingStatus.ErrorOccurred:
+                        return Resources.Status_ErrorOccurred;
+
+                    case LoadingStatus.NoMoreItems:
+                        return Resources.Status_NoMoreItems;
+
+                    case LoadingStatus.Ready:
+                        return Resources.Status_Ready;
+
+                    default:
```

Comment:
nit: should default be listed last, even if another case also is handled by the same code. (if others aren't bothered by this, don't stress)

Reply:
Changed and disabled other tests.